.buildkite/hooks/post-command

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -euo pipefail
```

Comment:
usual style is to separate these out.

Reply:
Do we not want -x

Reply:
ah just seen below. Do the two paths both get run?

Reply:
I kept the style from the pre-exit hook that we had before